onnxruntime/core/providers/qnn/builder/opbuilder/expand_op_builder.cc

```diff
@@ -131,6 +138,19 @@ Status ExpandOpBuilder::ProcessInputs(QnnModelWrapper& qnn_model_wrapper,
   return Status::OK();
 }
 
+Status ExpandOpBuilder::OverrideOutputQuantParam(QnnModelWrapper& qnn_model_wrapper,
+                                                 const NodeUnit& node_unit,
+                                                 const logging::Logger& logger,
+                                                 const std::vector<std::string>& input_names,
+                                                 size_t output_index,
+                                                 Qnn_DataType_t qnn_data_type,
+                                                 Qnn_QuantizeParams_t& quant_param) const {
+  // Force Expand output to use the same quantization parameters as the input if they are nearly equal.
+  // This enables the HTP backend to employ certain optimizations.
+  return SetOutputQParamEqualToInputIfNearlyEqual(qnn_model_wrapper, node_unit, logger, input_names,
```

Comment:
>SetOutputQParamEqualToInputIfNearlyEqual [](start = 9, length = 40)

For some ops we know it's safe to let the output share the same quantization parameters with input. Maybe we can directly for the output share the same with input?

Reply:
Always forcing them to be equal can affect accuracy because the quantization script will merge a Clip into a QuantizeLinear. Example: Reshape -> Clip will be converted into DQ -> Reshape -> Q', where Q' has different quantization parameters due to the original Clip.